### DISCUSSION TITLE
N-log 怎么高亮某些关键字（如error、info）

### DISCUSSION THREAD

**Discussion by dongpeeng** [tusen-ai/naive-ui#6083] [General]
N-log 怎么高亮某些关键字（如error、info）
                <NLog
                    rows={30}
                    log={this.logRef}
                    language='json'
                    hljs={hljs}
                    loading={this.logLoadingRef}
                    style={{
                        height: isFullscreen ? 'calc(100vh - 140px)' : '525px'
                    }}
                />

**Comment by Sepush** [comment]
https://www.naiveui.com/zh-CN/os-theme/components/log
阅读 log 组件的文档 和  highlight.js 的文档 对  highlight.js 进行设置
log 组件的文档中 定义了一个 naive-log 你也可以定义一个你的
